### PR TITLE
Scss.cards improvements

### PR DESCRIPTION
**Pull request by jeremie-lucca**
image

### CODE REVIEW

File: packages/scss/src/components/_card.scss [hunk 1]
```diff
@@ -29,4 +32,14 @@ $cardPadding: 1.5rem;
...
 		background: _theme("commons.background.color");
 	}
+
+	&.mod-clickable {
+		color: _color("text.default");
+		cursor: pointer;
+		text-decoration: none;
+
+		&:hover {
+			box-shadow: 0 8px 12px rgba(0, 0, 0, .15), 0 0 2px rgba(0, 0, 0, .3);
```
Comment (sanlucca) [inline packages/scss/src/components/_card.scss]: We might use a mixin to have consistant box-shadow changes over the whole framework

File: packages/scss/src/components/_list.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+.list-item-content-description {
+	margin: 0;
+}
+
+.list-item-actions {
+	margin-left: auto;
+	opacity: 0;
+	padding-left: 1rem;
+	transition: all 150ms ease;
```
Comment (sanlucca) [inline packages/scss/src/components/_list.scss]: Why not using the common.transition variable ?

File: packages/scss/src/components/_list.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.list-item-actions-icon {
+	background: transparent;
+	border: 0;
+	border-radius: 50%;
+	color: _color("text.default");
+	cursor: pointer;
+	height: 2rem;
+	transition: all 100ms ease;
```
Comment (sanlucca) [inline packages/scss/src/components/_list.scss]: Does it really need 50ms less than the common.transition variable ?

File: packages/scss/src/components/_card.scss (hunk 1 above)
Comment (jeremie-lucca) [inline packages/scss/src/components/_card.scss]: I putted it in commons.hover.box-shadow

File: packages/scss/src/components/_list.scss (hunk 2 above)
Comment (jeremie-lucca) [inline packages/scss/src/components/_list.scss]: done

File: packages/scss/src/components/_list.scss (hunk 3 above)
Comment (jeremie-lucca) [inline packages/scss/src/components/_list.scss]: done

**Pull request merged**